test/integration/worker/entity.js

```diff
@@ -872,6 +872,98 @@ describe('worker: entity', () => {
           person.currentVersion.version.should.equal(3);
         });
     }));
+
+    it('should update an entity and mark it SOFT conflict', testService(async (service, container) => {
```

Comment:
As I move tests around (moving many things out of this file in particular), I'm thinking about how entity update tests could end up in the right locations. These two tests are becoming, in essence, about `conflict` and `currentVersion.conflictingProperties` coming through the `GET /datasets/:name/entities/:uuid` endpoint, right? Maybe they could get moved there.